docs/UserGuide.md

```diff
@@ -203,14 +203,14 @@ Example: `jobdelete 2` deletes the second job
 
 ### Mark a job as paid : `mark`
 
-Indicates that a job has been completed.
-
-❗️Caution: This is irreversible.
+Indicates that a job has been completed. To unmark an object, `mark` the job again.
 
 Format: `mark JOBID`
 
 Example: `mark 2` marks the second job
 
+Continued Example: `mark 2` unmarks the second job
```

Comment:
"continued example" sounds a bit weird idk, maybe just elaborate on the original example?

> Example: `mark 2`
> Marks the second job if it isn't marked as paid, or unmarks it if [...]